lib/gooddata/helpers/global_helpers.rb

```diff
@@ -231,6 +231,8 @@ def encrypt(data, key)
       end
 
       def decrypt(database64, key)
+        vcr_mode_none = ENV['VCR_ON'] == 'true' && (!ENV['VCR_RECORD_MODE'] || ENV['VCR_RECORD_MODE'] == 'none') # TODO: refactor
```

Comment:
error: Identical code found in global_helpers.rb:234, connection_spec.rb:21 (mass = 68) (flay)